README.md

```diff
@@ -18,6 +18,8 @@ on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
 
+If this claim was actually true I would expect to see the run time be $O(n)$ for any data set I put in regardless of how it was previously sorted or what they had in them, where n is the number of inputs.
```

Comment:
What does that mean? How would you tell if a runtime is in $O(n)$?

Reply:
While increasing the size of the lists it sorts. If the amount of time it takes to sort increases linearly with the size of the list, one could infer that it is in fact sorting in linear time.

Reply:
Ok, then please say that.